crates/op-hardforks/src/lib.rs

```diff
@@ -8,11 +8,11 @@
 #![no_std]
 
 extern crate alloc;
-
+mod optimism;
 use alloc::vec::Vec;
 use alloy_hardforks::{hardfork, EthereumHardfork, EthereumHardforks, ForkCondition};
-mod optimism;
-use optimism::mainnet::*;
+use optimism::{mainnet::*, sepolia::*};
```

Comment:
```suggestion
pub use optimism::{mainnet::*, sepolia::*};
```
let's export these from crate root